packages/mdc-select/foundation.ts

```diff
@@ -185,10 +197,24 @@ export class MDCSelectFoundation extends MDCFoundation<MDCSelectAdapter> {
 
   handleMenuOpened() {
     this.adapter_.addClass(cssClasses.ACTIVATED);
+
+    if (this.adapter_.getMenuItemValues().length === 0) {
+      return;
+    }
+
+    // Menu should open to the last selected element, should open to first menu item otherwise.
+    const focusItemIndex = this.selectedIndex_ >= 0 ? this.selectedIndex_ : 0;
+    this.adapter_.focusMenuItemAtIndex(focusItemIndex);
   }
 
   handleMenuClosed() {
     this.adapter_.removeClass(cssClasses.ACTIVATED);
+    this.isMenuOpen_ = false;
+
+    this.adapter_.setSelectedTextAttr('aria-expanded', 'false');
+    if (!this.adapter_.isSelectedTextFocused()) {
+      this.handleBlur();
```

Comment:
We may not call handle* inside another handle* method in foundation. This might introduce problem in future if we add additional logic to calling handler method. I've seen this pattern in other places but this'll create issues in future.

Abstract required logic to separate method if required.

Reply:
I should have asked this earlier, but do you know this is even here? Is there any case in which the menu closed but the selected text is not focused?

Reply:
I think this is handling a case where menu is closed without a selection (By clicking outside the select or TAB'ing out). It removes the focused state from the component.

Reply:
Moved blur logic into private helper to be called in both handlers.